node-rfc/Dockerfile

```diff
@@ -21,17 +21,22 @@ COPY cts ${SAPBIN}/cts
 WORKDIR ${SAPBIN}
 
 RUN apt-get update && \
-    apt-get install -y apt-utils bzip2 unzip && \
-    rm -rf /var/lib/apt/lists/* /var/cache/apt/* && \
-    mkdir -p ${SAPDATADIR} && \
-    chown 1000:1000 ${SAPDATADIR} &&\
-    chmod a+x ${SAPBIN}/cts && \
-    cd ${SAPDIR} && \
-    unzip /tmp/$NWRFC_FILE && \
+    apt-get install -y apt-utils bzip2 unzip --no-install-recommends && \
+    rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+WORKDIR  ${SAPDATADIR} 
```

Comment:
Is the workdir statement here required? The two statement executed before the next WORKDIR are operating upon absolut directories. The workdir does not matter IMO.

Reply:
Yes it substitutes the mkdir statement.

Reply:
mmmhhh ... So we have one layer more only for creating a directory. What is bad with having the mkdir in the `RUN` above?

Reply:
When I remember correctly it was marked by the linter check. Otherwise I would have kept it.